src/package.ts

```diff
@@ -19,16 +24,22 @@ import PasswordReset from 'components/PasswordReset';
 import ThankYouPage from 'components/ThankYouPage';
 import Checkout from 'components/Checkout';
 import RedeemGift from 'components/RedeemGift';
-import eventDispatcher, {
-  MSSDK_PURCHASE_SUCCESSFUL
-} from 'util/eventDispatcher';
 
 if (typeof window !== 'undefined') {
   window.onload = () => {
     const queryString = window.location.search;
     const urlParams = new URLSearchParams(queryString);
     const externalPaymentId = urlParams.get('externalPaymentId');
-    if (externalPaymentId) {
```

Comment:
i think this change introduces a bug.

right now, the check for `externalPaymentId`'s presence got removed, so we will always send the `MSSDK_PURCHASE_SUCCESSFUL` event. Even, when the `externalPaymentId` is absent

please bring back this check, but leave the new one with the Mixpanel tracking

Reply:
No, the check is still present but now it also checks this `externalPaymentId` using `isPayPalExternalId(externalPaymentId)` function, so the check is still here and it will not send the event when `externalPaymentId` is absent.

Reply:
@m4tewoosh, right, sorry, my mistake. I didn't fully get that from the GH changes view.

However, is it possible that we have an `externalPaymentId` that is not PayPal? Like, after a 3DS redirect? If yes, then how the code works now is that it will only send the custom event for PayPal payments, and will cease to do so for others